Dockerfile

```diff
@@ -10,4 +10,4 @@ COPY . .
 
 EXPOSE 3000
 
-RUN npm run build
+# RUN npm run build
```

Comment:
Just realized we don't need to run this command, since in docker-compose we're running `npm run dev`, and not` npm run start`, which reads the files compiled in `npm run build`